rust/connlib/libs/client/src/messages.rs

```diff
@@ -113,6 +118,7 @@ pub enum EgressMessages {
         resource_id: Id,
         connected_gateway_ids: Vec<Id>,
     },
+    CreateLogSink {},
```

Comment:
@AndrewDryga How would I go about testing the serialisation of these messages? Could you give me a JSON snippet of what you expect and what you are going to send back?

Reply:
@thomaseizinger it actually returns a string which is a signed URL, there will be no JSON structure in the response.

Reply:
The reply message itself with all the metadata is still JSON though, right?

Reply:
Phoenix websocket messages are JSON encoded, here is an example:

```json
{
  "topic": "room:lobby",
  "ref": null,
  "payload": {
    "hello": "world"
  },
  "join_ref": null,
  "event": "shout"
}
```

My assumption is, that in this case, `payload` is a string and not a JSON object. I'll see if I can get the portal running today to confirm.

Reply:
FWIW, I didn't manage to create a working example locally because I couldn't figure out how to "enable" the message. But I think I reverse-engineered from the other messages as the following:

```json
{"event":"phx_reply","ref":"unique_log_sink_ref","topic":"client","payload":{"status":"ok","response":"https://storage.googleapis.com/foo/bar"}}
```